conformance/utils/kubernetes/helpers.go

```diff
@@ -977,3 +994,36 @@ func findPodConditionInList(t *testing.T, conditions []v1.PodCondition, condName
 	tlog.Logf(t, "%s was not in conditions list", condName)
 	return false
 }
+
+// BackendTLSPolicyMustHaveCondition checks that the created BackentTLSPolicy has the Condition,
+// halting after the specified timeout is exceeded.
+func BackendTLSPolicyMustHaveCondition(t *testing.T, client client.Client, timeoutConfig config.TimeoutConfig, policyNN, gwNN types.NamespacedName, condition metav1.Condition) {
+	t.Helper()
+	// Use HTTP Route timeout condition. This way we are sure that route for a given Service was applied.
+	waitErr := wait.PollUntilContextTimeout(context.Background(), 1*time.Second, timeoutConfig.HTTPRouteMustHaveCondition, true, func(ctx context.Context) (bool, error) {
+		route := &v1alpha3.BackendTLSPolicy{}
+		err := client.Get(ctx, policyNN, route)
+		if err != nil {
+			return false, fmt.Errorf("error fetching BackendTLSPolicy: %w", err)
+		}
+
+		for _, parent := range route.Status.Ancestors {
+			if err := ConditionsHaveLatestObservedGeneration(route, parent.Conditions); err != nil {
+				tlog.Logf(t, "BackendTLSPolicy %s (parentRef=%v) %v",
+					policyNN, parentRefToString(parent.AncestorRef), err,
+				)
+				return false, nil
+			}
+
+			if parent.AncestorRef.Name == gatewayv1.ObjectName(gwNN.Name) && (parent.AncestorRef.Namespace == nil || string(*parent.AncestorRef.Namespace) == gwNN.Namespace) {
```

Comment:
Just curious - why is it ok for the namespace to be nil here?

Reply:
According to the [spec](https://github.com/kubernetes-sigs/gateway-api/blob/0fd1805d4f97b4ee6a57339f3a411a26018ad699/apis/v1/shared_types.go#L60) this field is optional